.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: node_js
-node_js:
-  - "0.10"
-  - "4"
-  - "lts/*"
-  - "node"
-script: |
-  if [ `node --version | cut -d . -f 1,2` = "v0.10" ]; then
-    sed -i s/0o755/0755/ test/index.js;
-    npm test;
-  else
-    npm run test:lint && npm test;
-  fi
+
+jobs:
+  include:
+    - stage: spec tests 👩🏽‍💻
+      node_js: v0.10
```

Comment:
It looks like this ran properly but why is the first `node_js` line missing the `-` hyphen prefix?

Reply:
that is the same as json:

```json
[
      {
        "stage": "spec tests 👩🏽‍💻",
        "node_js": "v0.10"
      },
      {
        "node_js": "v4"
      }
]
```

the hyphen is the start of an array item.